distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -868,11 +868,12 @@ Date.from (that:JS_Object) =
         Error.throw (Illegal_Argument.Error "Invalid JS_Object for Date.from.")
 
 ## PRIVATE
-private make_day_picker cache=Nothing =
+private make_day_picker value cache =
+    _ = value
     year = cache.if_not_nothing <| cache "year"
     month = cache.if_not_nothing <| cache "month"
     days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
     max = if month.is_nothing then 31 else
         if month != 2 || year.is_nothing then days.at (month - 1) else
-            if Date.new year month 1 . is_leap_year then 29 else 28
+            if Date.new year 1 1 . is_leap_year then 29 else 28
```

Comment:
btw. why this change? It seems like both old and new code were doing the same thing anyway?

Reply:
Just didn't seem necessary to feed this into the Date.

Reply:
Fair